Porting/exec-bit.txt

```diff
@@ -3,69 +3,70 @@
 # 
 # List files (or glob patterns) one per line relative to the repo root
 # Lines starting with a pound and empty lines will be ignored
-Configure
-Makefile.SH
-Policy_sh.SH
```

Comment:
why so much noise in this file? seems like we are using a smarter sort :-)

Reply:
I just used "sort" from the shell. I can remove this patch and do it in a separate PR if you prefer. But in vim its really easy to just call sort on a list. :-)

Reply:
I dropped this commit. Ill do it as its own pr.

Reply:
no problem with it, was trying to understand what was happening and then realized it lol
fine as part of this change or as a later commit both work for me